package/views.py

```diff
@@ -254,13 +264,6 @@ def get_queryset(self):
         )
 
 
-class PackagePython3ListView(PackageSingleTableMixin):
-    template_name = "package/python3_list.html"
-
-    def package_filters(self):
-        return {"version__supports_python3": True}
-
-
```

Comment:
Why did we remove this?

Reply:
We had a merge conflict with the newer version (from my branch being too old). 

I'll double-check https://github.com/djangopackages/djangopackages/blob/e045bbc06fa36ae4f1d26a8539776db8709f6308/package/views.py#L455C7-L465 this afternoon.